src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,100 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+
+    private Pair<K, V>[] pairs = new Pair[MAX_SIZE];
+    private int size = 0;
```

Comment:
It is 0 by default. No need to set it 0
```suggestion
    private int size;
```